index.html

```diff
@@ -0,0 +1,127 @@
+          <li><img class="twiter" src="./images/twiter.png" alt="twiter logo"></li>
+          <li><a href="#" class="nav1-item">Amharic</a></li>
+          <li><a href="#" class="nav1-item">My Page</a></li>
+          <li><a href="#" class="nav1-item">Logout</a></li>
+        </ul>
+      </nav>
+    </section>
+    <section class="main-nav">
+      <a class="menu-link" href="#"><img src="./images/menu_FILL0_wght400_GRAD0_opsz48 (1).svg" alt=""></a>
+      <nav class="second-nav">
+        <div class="click-event">
+          <a class="logo-link" href="./index.html"><img class="logo-img" src="./images/logo.png" alt="logo"></a>
+          <div class="close-btn">&times;</div>
+          <ul class="nav2">
+            <li><a href="./about.html" class="menu-item text-red">About</a></li>
+            <li> <a href="./index.html#program" class="menu-item">Program </a></li>
+            <li> <a href="#" class="menu-item">Join</a></li>
+            <li> <a href="#" class="menu-item">Sponsor</a></li>
+            <li> <a href="#" class="menu-item">News</a></li>
+            <li> <a href="#" class="menu-item" id="campaign">GEM Campaign</a></li>
+          </ul>
+        </div>
+      </nav>
+    </section>
+  </header>
+   <main>
+    <section class="home">
+      <p class="openning">"Hello! Ethiopian Youth"</p>
+      <h1 class="title">
+        <span class="multicolor-text">Greate Ethiopian Motivational speakers 2023</span>
+      </h1>
+      <div class="desc-cont">
+        <p class="description">This is a real motivational Message for you! 
+          Are you looking for motivation for your life? New direction, happiness, success, contentment, new hope and strength?
+          Do you know who or what really motivates you in life and where your path is leading? 
+          If you need answer for these all questions please don't miss this programme which takes place in January, in Addisababa. 
+        </p>
+      </div>
+      <h2 class="date">2023.1.27(FRI) ~ 29(SUN)</h2>
+      <p class="place">@ Millinium hall, Art center Addisababa</p>
+    </section>
+    <section class="main-prog">
+      <h3 class="main-prog-title">Main Porgram</h3>
+      <div class="line"></div>
+      <div class="cont-flex-cont">
+        <div class="cont-flex">
+          <img class="prog-img" src="./images/opening.png" alt="Opening ceremony icon">
+          <p class="prog-title">Opening</p>
+          <p class="prog-desc">Introduction about the programme and message from the coordinators.</p>
+        </div>
+        <div class="cont-flex">
+          <img class="prog-img" src="./images/motivational-speech.png" alt="">
+          <p class="prog-title">Speech</p>
+          <p class="prog-desc">Motivational speakers from various 
+            backgrounds about the messages of self development and mindset.</p>
+        </div>
+        <div class="cont-flex">
+          <img class="prog-img" src="./images/like.png" alt="">
+          <p class="prog-title">Thanks</p>
```

Comment:
- [ ]  Your images should contain alt attributes with values in it, this will make your page more accessible with clear codes.